instrumentation/jaxrs-client/jaxrs-client-2.0-testing/build.gradle.kts

```diff
@@ -24,7 +24,7 @@ dependencies {
   latestDepTestLibrary("org.glassfish.jersey.inject:jersey-hk2:2.+")
   latestDepTestLibrary("org.glassfish.jersey.core:jersey-client:2.+")
   latestDepTestLibrary("org.jboss.resteasy:resteasy-client:3.0.26.Final")
-  latestDepTestLibrary("org.apache.cxf:cxf-rt-rs-client:3.6.5")
+  latestDepTestLibrary("org.apache.cxf:cxf-rt-rs-client:3.+")
```

Comment:
for some reason this was trying to pull in 3.6.6 last night, [which doesn't exist](https://mvnrepository.com/artifact/org.apache.cxf/cxf-rt-rs-client). I guess the issue was transitory, but im not sure why it would look for a version that doesn't exist?

Reply:
when there is a new release then sometimes issues like that happen